api/src/test/java/org/apache/iceberg/TestHelpers.java

```diff
@@ -137,6 +142,18 @@ public static void assertSerializedAndLoadedMetadata(Table expected, Table actua
     Assert.assertEquals("History must match", expected.history(), actual.history());
   }
 
+  public static void assertSerializedMetricsReporter(
```

Comment:
no need to add this here, this can be just part of the test method itself

Reply:
done